### ISSUE TITLE
Built-in projects Mkdocs Insiders Monorepo Plugin

### ISSUE THREAD

**Issue by jorgegomezcq** [backstage/mkdocs-monorepo-plugin#106]
Hi,

Will this project continue to be developed in light of the new Built-in projects plugin that is part of MkDocs Insiders?

https://squidfunk.github.io/mkdocs-material/setup/building-an-optimized-site/#built-in-projects-plugin

**Comment by pjungermann** [comment]
The `projects` plugin is "sponsors only" at this time. Some of the `edit_uri` logic might still be needed. 🤔

**Comment by squidfunk** [comment]
Author of Material for MkDocs here. We're really psyched giving the projects plugin into more hands and adding all functionality that is necessary to build documentation projects from multiple sources. I'm not sure what the mkdocs-monorepo plugin does with edit URIs, but the projects plugin allows to define a custom configuration transform function via the [`projects_config_transform`](https://squidfunk.github.io/mkdocs-material/plugins/projects/#config.projects_config_transform) setting that allows to customize the configuration of projects when build together.

[We're using it on our examples reposistory](https://github.com/mkdocs-material/examples/blob/master/examples/__init__.py) to customize `edit_uri` per project.

**Comment by bih** [comment]
We're aware of the projects plugin created by Material for Mkdocs, and it offers slightly different needs. They seem to have their plugin only available to paid customers (which is different from us) and their plugin seems tightly coupled with the use of Mkdocs Material. If those two things no longer become true, we'd happily sunset this plugin :)

**Comment by squidfunk** [comment]
The [projects plugin](https://squidfunk.github.io/mkdocs-material/plugins/projects/) works with all MkDocs projects and themes, so you're not bound to Material for MkDocs ☺️ Just use:

``` yaml
plugins:
  - material/projects
```

You can combine any MkDocs projects with any themes and mix and match.

Other than that, we have our plugin [available to sponsors](https://squidfunk.github.io/mkdocs-material/insiders/) that support our ongoing efforts in keeping Material for MkDocs and its plugins in good shape and up-to-date. We're not considering them customers, but users who actually care about Open Source and the people behind it, supporting us in trying to make the world of software a better place.

**Comment by bih** [comment]
@squidfunk Ah thanks for the clarification. Are there any intentions to make this available to all users, and not just sponsors who support you via $15/month?

**Comment by squidfunk** [comment]
Of course, as promised by the [sponsorware strategy](https://squidfunk.github.io/mkdocs-material/insiders/). It will be released into the community edition once the [associated funding goal](https://squidfunk.github.io/mkdocs-material/insiders/#24000-blockpaprika) is hit. We've and will always keep our promises.

**Comment by bih** [comment]
Oh fantastic! In that case, we're very open to the idea of sunsetting this library once the plugin is available to all users.
